source/reference/versioning.txt

```diff
@@ -0,0 +1,119 @@
+.. _c2c-release-version-numbers:
+
+===============================
+``mongosync`` Version Numbering
```

Comment:
This is a small thing, but "Version Numbering" seems far too narrow. I think we should just call this "Versioning" since the way the binary is versioned is more than just the numbering. For reference, the server also calls their equivalent page "Versioning": https://www.mongodb.com/docs/manual/reference/versioning/